setup.cfg

```diff
@@ -7,4 +7,3 @@ use_parentheses = True
 
 [mypy]
 ignore_missing_imports = True
-python_version = 3.4
```

Comment:
This is needed because CircuitPython is most equivalent to Python 3.4 in syntax/stdlib.